_posts/2018-09-04-atmos-ocean-launch.md

```diff
@@ -0,0 +1,74 @@
+associated with analyzing climate model data for a national science organization,
+so I was searching online for assistance with Python programming.
+I stumbled upon the website of an organization called Software Carpentry,
+which at the time was a relatively small group of volunteers
+running two-day scientific computing "bootcamps" for researchers.
+I reached out to ask if they'd be interested
+in running a workshop alongside the 2013 Annual Conference
+of the Australian Meteorological and Oceanographic Society (AMOS),
+and to my surprise Greg Wilson - the co-founder of the organization -
+flew out to Australia to teach at our event in Melbourne and another in Sydney
+(the first ever bootcamps outside of North America and Europe).
+I trained up as an instructor soon after,
+and from 2014-2017 I hosted Software Carpentry workshops alongside the AMOS conference,
+as well as other ad hoc workshops in various meteorology and oceanography departments.
+
+While these workshops were very popular and well received
+(Software Carpentry workshops always are),
+in the back of my mind I wanted to have a go at running a workshop
+designed specifically for atmosphere and ocean scientists.
+Instead of teaching generic skills
+in the hope that people would figure out how to apply them in their own context,
+I wanted to cut out the middle step and run a workshop
+*in* the atmosphere and ocean science context.
+This idea of discipline (or data-type) specific workshops
+was the driving force behind the establishment of Data Carpentry,
+so with their assistance I've developed lesson materials for a complete one-day workshop:  
+[https://carpentrieslab.github.io/python-aos-lesson](https://carpentrieslab.github.io/python-aos-lesson)
+
+The workshop centers around the task of writing a Python script
+that calculates and plots the seasonal rainfall climatology (i.e. the average rainfall)
+from the output from any arbitrary climate model.
+Such data is typically stored in netCDF file format
+and follows a strict "climate and forecasting" metadata convention.
+Along the way, we learn about the PyAOS stack
+(i.e. the ecosystem of libraries used in the atmosphere and ocean sciences),
+how to manage and share a software environment using conda,
+how to write modular/reusable code,
+how to write scripts that behave like other command line programs,
+version control,
+defensive programming strategies and
+how to capture and record the provenance of the data files and figures that we produce. 
+
+I've run the workshop twice this year
+(at the annual AMOS conference and at Woods Hole Oceanographic Institution),
+updating the lesson materials based on feedback each time,
+which means I've completed the alpha stage of the Data Carpentry
+[lesson development cycle](https://carpentries.github.io/curriculum-development/the-lesson-life-cycle.html).
+Before they can be adopted as the official lessons for atmosphere and ocean science,
```

Comment:
We're moving from alpha to beta, so the phrasing can be

"Moving from the alpha to beta development stage involves having people other than me teach."